Logger/FileLogger.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string? filePath;
```

Comment:
```suggestion
        private string _FilePath;
```
If you set this in the constructor & check for null there you don't have to mark it as nullable.